src/app/core/common-components/entity-subrecord/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -388,28 +385,6 @@ export class EntitySubrecordComponent<T extends Entity> implements OnChanges {
     row.formGroup = null;
   }
 
-  /**
-   * Delete the given entity from the database (after explicit user confirmation).
-   * @param row The entity to be deleted.
-   */
-  delete(row: TableRow<T>) {
-    this.entityRemoveService
-      .remove(row.record, {
-        deletedEntityInformation: $localize`:Record deleted info:Record deleted`,
-        dialogText: $localize`:Delete confirmation message:Are you sure you want to delete this record?`,
-      })
-      .subscribe((result) => {
-        switch (result) {
-          case RemoveResult.REMOVED:
-            this.removeFromDataTable(row.record);
-            break;
-          case RemoveResult.UNDONE:
-            this.records.unshift(row.record);
-            this.initFormGroups();
```

Comment:
this is (nowadays) already happening through the `listenToEntityUpdates()` anyway.

Reply:
Good find